### PR TITLE
Implemented a single columned list for the kubernetes metrics reference page

### LINKED ISSUE #42342: Improve styling of metrics reference
**This is a Feature Request**

**What would you like to be added**

**_Without_** changing https://kubernetes.io/docs/reference/instrumentation/metrics/ itself, edit the SCSS for this website to improve how that page appears. The column sizes don't render well right now, especially on narrow viewports.

**Why is this needed**
The column sizes don't render well right now, especially on narrow viewports.

**Comments**
You can find the files to edit inside https://github.com/kubernetes/website/tree/main/assets/scss

/area web-development
/help
/triage accepted
/priority backlog

### PR DESCRIPTION
**Pull request by Gauravpadam**
#### This PR closes #42342 and #38438

As #38438 demands a generator code change, I've changed the generator codes for the metrics.md
(No manual changes have been made to the metrics.md rather than the ones generated by the generator itself!)
I'll be linking the PR from k8s/k8s below once I finish making it
(Edit) The pull request for generator code change: https://github.com/kubernetes/kubernetes/pull/120350
(Edit) https://github.com/kubernetes/kubernetes/pull/120350 has been merged successfully

Appropriate styling changes have been made as per the generated html, As requested in #42342

### CODE REVIEW

**Comment by sftim** [comment]
/hold

pending merge of https://github.com/kubernetes/kubernetes/pull/120350

File: assets/scss/_custom.scss [hunk 1]
```diff
@@ -349,52 +349,62 @@ footer {
...
-  }
 
-/* SCSS Related to the Metrics Table */
+/* SCSS Related to the Metrics list */
+
+  div.metric:nth-of-type(odd) { // Look & Feel , Aesthetics
+    background-color: $light-grey;
+  }
 
-    @media (max-width: 767px) { // for mobile devices, Display the names, Stability levels & types
```
Comment (tengqm) [inline assets/scss/_custom.scss:359]: Is this breaking the mobile view?

Comment (Gauravpadam) [reply assets/scss/_custom.scss:359]: This was necessary when we had the old tabular format @tengqm, Now It just looks fine without it

**Comment by Gauravpadam** [comment]
@sftim I'm seeing some merge conflicts on metrics.md, Should we be changing anything in other PRs as well?

**Comment by dgrisonnet** [comment]
The new format looks awesome, thank you @Gauravpadam! :smiley:

**Comment by Gauravpadam** [comment]
> The new format looks awesome, thank you @Gauravpadam! 😃

Thanks @dgrisonnet, Glad you liked it! 😃

**Comment by Gauravpadam** [comment]
> @sftim I'm seeing some merge conflicts on metrics.md, Should we be changing anything in other PRs as well?

See github.com/kubernetes/website/pull/42879

**Comment by Gauravpadam** [comment]
@tengqm @sftim We're good to merge this PR, Both https://github.com/kubernetes/kubernetes/pull/120350 and https://github.com/kubernetes/kubernetes/pull/120369 have been merged successfully into k8s/k8s

**Comment by tengqm** [comment]
“unassigned" ... ;)

**Comment by sftim** [comment]
Thanks for the review @tengqm!

**Comment by tengqm** [comment]
/hold cancel
Good to go now.
/lgtm

**Comment by Gauravpadam** [comment]
Thanks for the reviews @sftim @tengqm!

**Comment by reylejano** [comment]
Preview: https://deploy-preview-42823--kubernetes-io-main-staging.netlify.app/docs/reference/instrumentation/metrics/

/approve

**Comment by sftim** [comment]
This was reverted in #44281 and would be reinstated by https://github.com/kubernetes/website/pull/44286

**Pull request merged**